src/app/stacktrace_win.h

```diff
@@ -221,11 +287,17 @@ const QString straceWin::getBacktrace()
                     fileName = fileName.mid(slashPos + 1);
             }
             QString funcName;
+            QString sourceFile;
             if(SymFromAddr(hProcess, ihsf.InstructionOffset, &dwDisplacement, pSymbol)) {
                 funcName = QString(pSymbol->Name);
 #ifdef __MINGW32__
                 demangle(funcName);
 #endif
+
+                // now ihsf.InstructionOffset points to the instruction that follows CALL instuction
+                // decrease the query address by one byte to point somewhere in the CALL instruction byte sequence
+                auto addr = reinterpret_cast<DWORD64>(reinterpret_cast<char*>(ihsf.InstructionOffset) - 1);
```

Comment:
I don't fully understand this.
1. Why reinterpret_cast to char* then -1 instead of `ihsf.InstructionOffset -1` (in type of ULONG64) directly?
2. Or cast to `uintptr_t`, is this better?
3. After -1 operation, why cast to `DWORD64`? why not `ULONG64` till the end?